pandas/io/formats/format.py

```diff
@@ -946,7 +947,7 @@ def _format(x):
             if self.na_rep is not None and is_scalar(x) and isna(x):
                 if x is None:
                     return 'None'
-                elif x is NaT:
+                elif x is NaT or is_np_nat(x):
```

Comment:
use is_null_datetimelike instead of rolling your own

Reply:
`is_null_datetimelike` is too permissive here (accepts `np.nan` and `None`)

Reply:
I don't want yet *another* NaT detection routine.

Reply:
`is_null_datetimelike` doesn't _just_ detect NaT though. It just detects null values (None, np.nan, NaT, iNaT)